src/Components/Members/MembersCreateEdit.js

```diff
@@ -0,0 +1,316 @@
+      <div className="thumb-inner">
+        <img className="img" src={file.preview} />
+      </div>
+    </div>
+  ));
+
+  useEffect(
+    () => () => {
+      // Make sure to revoke the data uris to avoid memory leaks
+      files.forEach((file) => URL.revokeObjectURL(file.preview));
+    },
+    [files],
+  );
+
+  const descriptionChange = (e, editor) => {
+    const data = editor.getData();
+
+    setInitialValues({ ...initialValues, description: data });
+  };
+  const nameChange = (e, values) => {
+    // eslint-disable-next-line no-console
+    console.log(values);
+    setInitialValues({ ...initialValues, name: e.target.value });
+  };
+  const facebookChange = (e) => {
+    setInitialValues({ ...initialValues, facebookUrl: e.target.value });
+  };
+  const linkedInChange = (e) => {
+    setInitialValues({ ...initialValues, linkedinUrl: e.target.value });
+  };
+  const handleClick = () => {
+    const datosConEtiquetas = initialValues.description;
+    const datosSinEtiquetas = datosConEtiquetas.replace(/<[^>]+>/g, '');
+
+    if (!id) {
+      axios
+        .post(urlCreate, {
+          name: initialValues.name,
+          image: initialValues.image,
+          description: datosSinEtiquetas,
+          facebookUrl: initialValues.facebookUrl,
+          linkedinUrl: initialValues.linkedinUrl,
+        })
+        .then((response) => {
+          if (response.status === 200) {
+            // eslint-disable-next-line no-console
+            alert('Create member successfully');
+
+            return initialValues;
+          }
+        })
+        // eslint-disable-next-line no-console
+        .catch((e) => console.log(e));
+    }
+    if (id) {
+      axios
+        .post(urlEdit, {
+          name: initialValues.name,
+          image: initialValues.image,
+          description: datosSinEtiquetas,
+          facebookUrl: initialValues.facebookUrl,
+          linkedinUrl: initialValues.linkedinUrl,
+        })
+        .then((response) => {
+          if (response.status === 200) {
+            // eslint-disable-next-line no-console
+            alert('Edit member successfully');
+
+            return setInitialValues(initialValues);
+          }
+        })
+        .catch((e) => alert(e));
+    }
+  };
+
+  useEffect(() => {
+    const dataEdit = async () => {
+      if (id) {
+        const initialData = await axios.get(urlEdit),
+          { name, image, description, facebookUrl, linkedinUrl } =
+            initialData.data.data;
+
+        if (name) setInitialValues({ ...initialValues, name: name });
+        if (image) setInitialValues({ ...initialValues, image: image });
+        if (description)
+          setInitialValues({ ...initialValues, description: description });
+        if (facebookUrl)
+          setInitialValues({ ...initialValues, facebookUrl: facebookUrl });
+        if (linkedinUrl)
+          setInitialValues({ ...initialValues, linkedinUrl: linkedinUrl });
+      }
+    };
```

Comment:
please move this to a function outside the useEffect. Also dataEdit do not describe what is being done here. Please update the name to something like loadMemberInfoOnEdit